deploy/warehouse/target/data.sql

```diff
@@ -27,16 +49,66 @@ insert into warehouse.organism (lineage, identifiers, details)
 
 with target_lineage (identifier, lineage) as (
     values
-        ('B.pertussis', 'Bordetella_pertussis'::ltree),
-        ('Flu_A_H1',    'Influenza.A.H1N1'),
-        ('Flu_A_H3',    'Influenza.A.H3N2'),
-        ('Flu_A_pan',   'Influenza.A'),
-        ('Flu_B_pan',   'Influenza.B'),
-        ('AP324NU',     'Influenza.C'),
-        ('Measles',     'Measles'),
-        ('Mumps',       'Mumps'),
-        ('RSVA',        'RSV.A'),
-        ('RSVB',        'RSV.B')
+        ('Adenovirus_pan_1',            'Adenovirus'::ltree),
+        ('Adenovirus_pan_2',            'Adenovirus'),
+        ('AdV_1of2',                    'Adenovirus'),
+        ('AdV_2of2',                    'Adenovirus'),
+        ('AI20U8U',                     'Bordetella_pertussis'),
+        ('B.pertussis',                 'Bordetella_pertussis'),
+        ('B.Pertussis',                 'Bordetella_pertussis'),
+        ('AI1RW2H',                     'Chlamydophila_pneumoniae'),
+        ('C_pneumoniae',                'Chlamydophila_pneumoniae'),
+        ('C. pneumoniae',               'Chlamydophila_pneumoniae'),
+        ('C. pneumoniae_AI1RW2H',       'Chlamydophila_pneumoniae'),
+        ('C.pneumoniae',                'Chlamydophila_pneumoniae'),
+        ('AP7DPVF',                     'Enterovirus'),
+        ('EnterovirusA_B 1_AP7DPVF',    'Enterovirus'),
+        ('Enterovirus_pan',             'Enterovirus'),
+        ('Ev_pan',                      'Enterovirus'),
+        ('EV_pan',                      'Enterovirus'),
```

Comment:
Based on the target name `EnterovirusA_B 1_AP7DPVF` and the target equivalency spreadsheet on the Trello card, I think these all actually target only Enterovirus A and B. This gives them the same problem as the Coronavirus combos.

Reply:
I think this applies not just to `EnterovirusA_B 1_AP7DPVF` but also the three variations on the pan Enterovirus targets. The old target name was `Enterovirus_pan` which is now `EnterovirusA_B 1_AP7DPVF`, but I don't think the actual target changed, just the name...

Reply:
I've asked Peter for confirmation: https://seattle-flu-study.slack.com/archives/CCAEWSFTK/p1570490686074300

Reply:
Oh, I see now. Thanks for checking with Peter.

Reply:
Ah, looks like the A and B assumption was wrong:

> It does not just pick up the A and B subtypes.  That assay ID AP7DPVF includes all enterovirus species A, B, C, D, G

So we can leave this as-is!